openmrs/src/test/java/org/motechproject/openmrs/resource/impl/ProgramEnrollmentResourceImplTest.java

```diff
@@ -77,6 +81,25 @@ public void shouldCreateProgramEnrollment() throws Exception {
     }
 
     @Test
+    public void shouldCreateProgramEnrollmentWithAttributes() throws Exception {
+        ProgramEnrollment programEnrollment = prepareBahmniProgramEnrollment();
+
+        URI url = config.toInstancePath("/bahmniprogramenrollment");
+
+        doReturn(getResponseFromFile(BAHMNI_PROGRAM_ENROLLMENT_RESPONSE)).when(restOperations)
+                .exchange(eq(url), eq(HttpMethod.POST), any(HttpEntity.class), eq(String.class));
+
+        ProgramEnrollment created = programEnrollmentResource.createBahmniProgramEnrollment(config, programEnrollment);
+
+        verify(restOperations).exchange(eq(url), eq(HttpMethod.POST), requestCaptor.capture(), eq(String.class));
+
+        assertThat(created, equalTo(programEnrollment));
```

Comment:
ProgramEnrollment does not have attributes in equals method, so this check does not verify attributes